package.json

```diff
@@ -23,11 +24,51 @@
     "type": "git",
     "url": "https://github.com/FromDoppler/Doppler-UI-System"
   },
+  "browserslist": [
+    "last 2 versions",
+    "> 1%",
+    "last 3 iOS versions",
+    "Firefox > 20",
+    "ie 9"
+  ],
+  "webfont": {
+    "files": "./assets/icons/*.svg",
+    "fontName": "icon-font",
+    "templateClassName": "icon",
+    "templateFontPath": "../fonts/",
+    "template": "./assets/scss/libs/webfont/template.css"
+  },
   "scripts": {
-    "semantic-release": "semantic-release",
+    "copyfiles": "./node_modules/.bin/cpy",
+    "css-compile-dist": "npm run font && npm run --silent css-compile-sass > css-prefix > ./dist/css/styles.css && npm run css-minify-dist",
+    "css-compile-dist-all": "npm-run-all css-compile-dist css-compile-dist-editor css-compile-dist-dark",
+    "css-compile-dist-dark": "npm run --silent css-compile-sass-dark > css-prefix > ./dist/css/theme-dark.css && npm run css-minify-dist-dark",
+    "css-compile-dist-editor": "npm run --silent css-compile-sass-editor > css-prefix > ./dist/css/editor.css && npm run css-minify-dist-editor",
+    "css-compile-sass": "node-sass --importer node_modules/node-sass-magic-importer/dist/cli.js ./assets/scss/styles.scss",
+    "css-compile-sass-dark": "node-sass --importer node_modules/node-sass-magic-importer/dist/cli.js ./assets/scss/themes/theme-dark/theme-dark.scss",
+    "css-compile-sass-editor": "node-sass --importer node_modules/node-sass-magic-importer/dist/cli.js ./assets/scss/themes/editors/editors.scss",
+    "css-minify-dist": "./node_modules/.bin/cleancss ./dist/css/styles.css -o ./dist/css/styles.css",
+    "css-minify-dist-dark": "./node_modules/.bin/cleancss ./dist/css/theme-dark.css -o ./dist/css/theme-dark.css",
+    "css-minify-dist-editor": "./node_modules/.bin/cleancss ./dist/css/editor.css -o ./dist/css/editor.css",
+    "css-prefix": "postcss --use autoprefixer --use postcss-flexibility",
+    "del-dist": "del-cli ./dist",
+    "dist": "npm-run-all del-dist font-dist makedir-css-dist css-compile-dist-all imagemin-dist html-dist js-minify-dist js-copy-push-dist sassdoc-dist",
+    "eclint-check": "./node_modules/.bin/eclint check \"**/*\"",
+    "font": "make-dir ./assets/scss/libs/webfont/compiled && webfont ./assets/icons/*.svg -d ./assets/fonts -s ./assets/scss/libs/webfont/compiled --font-height 1001 --normalize true",
+    "font-dist": "npm run font && npm run copyfiles ./assets/fonts/*.* ./dist/fonts",
+    "html-dist": "node ./assets/process-html.js dist",
+    "html-processor": "./node_modules/.bin/htmlprocessor",
+    "imagemin": "./node_modules/.bin/imagemin",
+    "imagemin-dist": "imagemin ./assets/img/* --out-dir ./dist/img",
+    "js-copy-push-dist": "npm run copyfiles ./assets/firebasePush.js ./assets/firebase-messaging-sw.js ./dist",
+    "js-minify": "./node_modules/.bin/uglifyjs ./assets/js/*.js --compress --mangle",
+    "js-minify-dist": "make-dir ./dist/js && npm run --silent js-minify > ./dist/js/app.js",
```

Comment:
Just asking from the ignorance point of view. Isn't there any fancy way of doing all these tasks without having to declare a regular one vs the dist(ributable) one? I'm quite rusty but maybe this has been improved in the recent years?

Reply:
I'll check, @matiasbeckerle thanks a lot for you feedback!

Reply:
Done!

Reply:
Niiice, thanks for investigating this!